server/services/case-service.js

```diff
@@ -31,25 +31,30 @@ const createCaseService = apiUrl => {
       const res = await request(`${apiUrl}/cases/${caseId}`)
       return res.data
     },
-    getMatchDetails: async defendantId => {
-      const res = await request(
+    getMatchDetails: async (defendantId, showAllMatches) => {
+      showAllMatches = Boolean(showAllMatches)
```

Comment:
this is not required - you have done this coercion before calling getMatchDetails